benchmark/MIcrosoft.IdentityModel.Benchmarks/prbenchmarks.identitymodel.config.yml

```diff
@@ -0,0 +1,22 @@
+﻿# multi-line script to build the project
+components:
+    jwt:
+        script: |
+            call dotnet build -c release --framework net8.0 .\src\Microsoft.IdentityModel.JsonWebTokens\Microsoft.IdentityModel.JsonWebTokens.csproj
+
+        arguments:
+            --application.options.outputFiles .\src\Microsoft.IdentityModel.JsonWebTokens\bin\release\net8.0\
+
+# default arguments that are always used on crank commands
+defaults: --config https://github.com/aspnet/Benchmarks/blob/main/build/azure.profile.yml?raw=true --application.framework net7.0  
```

Comment:
net7?

Reply:
Not the correct profile, use the one I suggested